app/controllers/ctc/questions/questions_controller.rb

```diff
@@ -2,6 +2,8 @@ module Ctc
   module Questions
     class QuestionsController < ::Questions::QuestionsController
       helper_method :wrapping_layout
+      skip_before_action :unauthenticated_redirect_in_offseason
```

Comment:
Will this code prevent clients from signing in? `Ctc::Portal::BaseAuthenticatedController` may need the same treatment.

Reply:
Only if this before action happens _before_ we look to see if someone is authenticated and redirect them. My thought was that would already be happening at a higher level.